### PR TITLE
[Task#69516] Add notification and logic for assign staff

### PR DESCRIPTION
**Pull request by tungbd-0214**
## Related Tickets:
- [#69516](https://edu-redmine.sun-asterisk.vn/issues/69516)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by tungbd-0214** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -256,13 +273,125 @@ public function detailOrderPage(Order $order)
             $product->creation_time = Carbon::create($product->pivot->created_at)->format('d/m/y H:i');
 
             if (isset($product->pivot->staff_id)) {
-                $staff = User::where('id', '=', $product->pivot->staff_id)->get('last_name');
-                $product->staff_name = $staff[0]->last_name;
+                $staff = User::where('id', $product->pivot->staff_id)->get();
+                $product->staff_name = $staff[0]->last_name." ".$staff[0]->first_name;
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]:
```suggestion
                $product->staff_name = $staff[0]->last_name . " " . $staff[0]->first_name;
```

File: app/Notifications/OrderProductNotification.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+                    ->line('The introduction to the notification.')
+                    ->action('Notification Action', url('/'))
+                    ->line('Thank you for using our application!');
+    }
```
Comment (ghost) [inline app/Notifications/OrderProductNotification.php]: indent ?

File: resources/js/Pages/orders/Show.jsx [hunk 3]
```diff
@@ -27,6 +57,35 @@ export default function Index(props) {
...
+                                label: 'In Process',
+                                value: 'In Process'
+                            },
+                            {
+                                label: 'Done',
+                                value: 'Done'
+                            },
+                            {
+                                label: 'Cancel',
+                                value: 'Cancel'
```
Comment (ghost) [inline resources/js/Pages/orders/Show.jsx]: localization

**Comment by tungbd-0214** [comment]
ready

**Pull request merged**